includes/sdk-ts-browser/tracking-an-event.md

```diff
@@ -12,3 +12,14 @@ const eventProperties = {
 };
 amplitude.track('Button Clicked', eventProperties);
 ```
+
+You can also pass an event object to the `track` function. Refer to [this link](https://amplitude.github.io/Amplitude-TypeScript/interfaces/_amplitude_analytics_browser.Types.BaseEvent.html) to see what you can include in the event object.
+
+```ts
+const event_propertie = {
+  buttonColor: 'primary',
+};
+
+const event = {event_type: "Button Clicked", event_properties};
```

Comment:
I think we should include another field here to show that it is used to set more that just event_type and event_properties. Maybe groups would be a good option?